ynput/ayon-core
# Nukestudio: Crash if save as new version with workfile - AY-854

## Changelog Description
Fix for edge case of saving workfile in NukeStudio that would crash the application.

## Testing notes:
1. Launch NukeStudio on task without workfiles.
2. `OpenPype -> Work Files -> Save As` and save.
3. Double click the newly created file and say 'no' to the saving question.
4. Confirm NukeStudio does not crash.
